public/js/monitor.js

```diff
@@ -25,42 +26,21 @@ function doXHR(aURL, aBodyObj, aAlertText, aDebug=true) {
   });
 }
 
-// eslint-disable-next-line no-unused-vars
-function addUser() {
-  doXHR("/user/add",
-        { email: document.getElementById("addUserField").value })
-    .then(function() {
-      alert("A verification link has been emailed to the specified address.");
-    });
+function isValidEmail(val) {
+  // https://stackoverflow.com/a/46181
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return re.test(String(val).toLowerCase());
 }
 
-// eslint-disable-next-line no-unused-vars
-function removeUser() {
-  doXHR("/user/remove",
-        { email: document.getElementById("removeUserField").value });
+function enableBtnIfEmailValid(e) {
+  const emailBtn = document.getElementById("subscribe-email-btn");
+  if (isValidEmail(e.target.value)) {
+    emailBtn.disabled = false;
+  } else {
+    emailBtn.disabled = true;
+  }
 }
-
-// eslint-disable-next-line no-unused-vars
-// function doOauth() {
-//   window.open("/oauth/init");
-// }
-
-// function isValidEmail(val) {
-//   // https://stackoverflow.com/a/46181
-//   const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-//   return re.test(String(val).toLowerCase());
-// }
-
-// function enableBtnIfEmailValid(e) {
-//   const emailBtn = document.getElementById("subscribe-email-btn");
-//   if (isValidEmail(e.target.value)) {
-//     emailBtn.disabled = false;
-//   } else {
-//     emailBtn.disabled = true;
-//   }
-// }
-
-
+*/
```

Comment:
Why not just ✂️ the code instead of commenting it out?
Or are we planning on resurrecting it shortly?

Reply:
We will likely bring it back right after the shield study runs.

Reply:
I'd almost argue we should have some "shield-study" branch where we'll deploy from, and then we can keep "master" branch less commenty.